### PR TITLE
fix pot tag

### PR DESCRIPTION
**Pull request by M-Rb3**
closes [#451](https://github.com/PotLock/bos-app/issues/451)

### CODE REVIEW

File: apps/potlock/widget/Pots/Card.jsx [hunk 1]
```diff
@@ -116,8 +116,8 @@ const applicationNotStarted = now < application_start_ms;
 const applicationOpen = now >= application_start_ms && now < application_end_ms;
 const publicRoundNotStarted = now < public_round_start_ms;
 const publicRoundOpen = now >= public_round_start_ms && now < public_round_end_ms;
-const publicRoundClosed = now >= public_round_end_ms && cooldown_end_ms;
-const cooldownOpen = publicRoundClosed && now < cooldown_end_ms;
+const publicRoundClosed = now >= public_round_end_ms && now > cooldown_end_ms;
```
Comment (lachlanglen) [inline apps/potlock/widget/Pots/Card.jsx:119]: this should just be `now >= public_round_end_ms`

Comment (lachlanglen) [inline apps/potlock/widget/Pots/Card.jsx:119]: this should just be `now >= public_round_end_ms`

**Review by lachlanglen — CHANGES_REQUESTED** [review]

File: apps/potlock/widget/Pots/Card.jsx [hunk 2]
```diff
@@ -116,8 +116,8 @@ const applicationNotStarted = now < application_start_ms;
...
+const cooldownOpen = now >= public_round_end_ms && now < cooldown_end_ms;
```
Comment (lachlanglen) [inline apps/potlock/widget/Pots/Card.jsx:120]: will this error if `cooldown_end_ms` is `null`?

File: apps/potlock/widget/Pots/Card.jsx (hunk 1 above)
Comment (M-Rb3) [reply apps/potlock/widget/Pots/Card.jsx:119]: If I did not add  now > cooldown_end_ms, the closed will appear with the cooldown tag

File: apps/potlock/widget/Pots/Card.jsx (hunk 2 above)
Comment (M-Rb3) [reply apps/potlock/widget/Pots/Card.jsx:120]: no, it will return false

**Pull request closed without merging**